packages/map-template/src/components/Search/Search.jsx

```diff
@@ -132,6 +132,11 @@ function Search({ onSetSize, isOpen }) {
         setSearchResults(locations);
         setFilteredLocations(locations);
         setShowNotFoundMessage(locations.length === 0);
+
+        // When category changes, we need to update scroll buttons enabled/disabled states.
```

Comment:
I think readers deserve to know why we grab hold of the `transitionend` event instead of doing it right away.

Reply:
✅